src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -36,6 +36,7 @@
 public class EditCommand extends Command {
 
     public static final String COMMAND_WORD = "edit";
+    public static final String UNARCHIVE_COMMAND_WORD = "unarchive";
```

Comment:
Same issue as this [https://github.com/AY2425S1-CS2103T-T13-2/tp/pull/98#discussion_r1820027896](https://github.com/AY2425S1-CS2103T-T13-2/tp/pull/98#discussion_r1820027896)